ec/resources/rulesengine/v4policy.go

```diff
@@ -71,6 +71,18 @@ func ResourcePolicyCreate(
 	policyMap := make(map[string]interface{})
 	json.Unmarshal([]byte(policy), &policyMap)
 	policyMap["state"] = "locked"
+	if policyMap["name"] != nil {
+		log.Printf(`
+		[WARN] Please remove policy name, "%s" in the policy.
+		It will be ignored. 
+		The policy name would be auto-generated by the provider with following format.
+		tf-policy-[customer_account_id]-[platform]-[deploy_to]-[timestamp]`,
+			policyMap["name"])
+	}
+	policyMap["name"] = fmt.Sprintf("tf-policy-%s-%s-%s-%d",
+		d.Get("account_number").(string),
+		d.Get("deploy_to").(string), policyMap["platform"],
+		time.Now().Unix())
```

Comment:
utc time perhaps?

Reply:
good point!

Reply:
I used `time.Now().Format(time.RFC3339)` for the delete operation... that might be good? I picked it because it's human readable since it will be in the UI.

Reply:
updated as time.Now().UTC().Format(time.RFC3339)